src/components/editor/faultTree/menu/faultEvent/FaultEventMenu.tsx

```diff
@@ -268,7 +268,7 @@ const FaultEventMenu = ({ shapeToolData, onEventUpdated, refreshTree, rootIri }:
       {/* ROOT NODE */}
       {shapeToolData && shapeToolData.iri === rootIri && (
         <>
-          <Typography>
+          <Typography className={classes.labelRow}>
```

Comment:
@Kasmadei, I do not understand this change. You are modifying event properties, and there are rows that contain:
label and value

Why do you call it `labelRow`? (not sure if you defined it or not, but it is not understandable to me)
 
The next line of code has "Event name" ... should be translated (i18n)